diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index b7cf765..cfcf437 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -32,6 +32,7 @@ const getClientEnvironment = require('./env');
 const ModuleNotFoundPlugin = require('react-dev-utils/ModuleNotFoundPlugin');
 const ForkTsCheckerWebpackPlugin = require('react-dev-utils/ForkTsCheckerWebpackPlugin');
 const typescriptFormatter = require('react-dev-utils/typescriptFormatter');
+const WebWorkerPlugin = require('@remote-ui/web-workers/webpack')
 // @remove-on-eject-begin
 const eslint = require('eslint');
 const getCacheIdentifier = require('react-dev-utils/getCacheIdentifier');
@@ -441,6 +442,7 @@ module.exports = function(webpackEnv) {
                         },
                       },
                     },
+                    require.resolve('@remote-ui/web-workers/babel')
                   ],
                 ],
                 // This is a feature of `babel-loader` for webpack (not Babel itself).
@@ -580,6 +582,7 @@ module.exports = function(webpackEnv) {
       ],
     },
     plugins: [
+      new WebWorkerPlugin(),
       // Generates an `index.html` file with the <script> injected.
       new HtmlWebpackPlugin(
         Object.assign(
